site/content/extend/plugins/overview.md

```diff
@@ -15,7 +15,7 @@ A more detailed example is the [Demo Plugin](https://github.com/mattermost/matte
 ### Manifest
 The plugin manifest provides required metadata about the plugin, such as name and ID. It is defined in JSON or YAML. This is `plugin.json` in both the [sample](https://github.com/mattermost/mattermost-plugin-starter-template/blob/master/plugin.json) and [demo](https://github.com/mattermost/mattermost-plugin-demo/blob/master/plugin.json) plugins.
 
-See the [manifest reference](/extend/plugins/manifest-reference/) for more information.
+See the [manifest reference](https://developers.mattermost.com/extend/plugins/manifest-reference/) for more information.
```

Comment:
I'm curious as to why this was changed? The existing link is accurate as it references a file in this section. The full URL is thus not needed.